docs/custom_framework.md

```diff
@@ -1,39 +1,39 @@
 ## 🎎 Custom Framework Config
 
-If you are using an i18n framework that does not have built-in support by this extension or you are using a custom i18n implementation, you can always make this extension support for your framework.
+If you are using an i18n framework that does not have built-in support by this extension, or you are using a custom i18n implementation, you can always make this extension support for your framework.
 
 Create file `.vscode/i18n-ally-custom-framework.yml` then copy and paste the configs below.
 
 ```yaml
 # .vscode/i18n-ally-custom-framework.yml
 
-# An array of string which contains Language Ids defined by vscode
+# An array of strings which contain Language Ids defined by VS Code
 # You can check avaliable language ids here: https://code.visualstudio.com/docs/languages/overview#_language-id
 languageIds:
   - javascript
   - typescript
   - javascriptreact
   - typescriptreact
 
-# An Array of regex to find the keys usage. **The key should captured in the first match group**.
-# You should unescape regex string in order to fit in YAML file
-# for that, you can use https://www.freeformatter.com/json-escape.html
+# An array of RegExs to find the key usage. **The key should be captured in the first match group**.
+# You should unescape RegEx strings in order to fit in the YAML file
```

Comment:
```suggestion
# An array of RegExps to find key usage. **The key should be captured in the first match group**.
# You should unescape RegExp strings in order to fit in the YAML file
```
I am afraid that `RegExs` looks kinda wired. Or maybe it shouldn't be plural? Not sure how others do it.

Reply:
According to [Wiktionary](https://en.wiktionary.org/wiki/regex), the plural of "regex" is actually "regexes". Would that look better? It should be plural, technically, since the array can contain more than one.

Reply:
Ok let's go for it.

Reply:
Done.